Makefile

```diff
@@ -15,8 +15,6 @@
 # Various versions - Minishift, default OpenShift, default B2D ISO
 MINISHIFT_VERSION = 1.3.1
 OPENSHIFT_VERSION = v1.5.1
-B2D_ISO_VERSION = v1.0.2
-CENTOS_ISO_VERSION = v1.1.0
```

Comment:
@hferentschik  Trying to understand your suggestion. Is it because we need a version to fall back in-case semantic version  check fails? Also what about the versions we put in documentation?